examples/tests/requirements.txt

```diff
@@ -10,3 +10,4 @@ torchvision==0.10.0
 pandas
 tensorflow_datasets
 scipy
+pillow<10.0.0
```

Comment:
Could you please leave a short comment explaining this pin and when we can remove it?